Model/Request/Buyers.php

```diff
@@ -160,11 +159,11 @@ protected function getParams()
         $params = [
             'topic' => $this->_topic
         ];
-
-        $params['firstname'] = $this->session->getCustomer()->getFirstname() ?? '';
-        $params['lastname'] = $this->session->getCustomer()->getLastname() ?? '';
-        $params['email'] = $this->session->getCustomer()->getEmail() ?? '';
-
+        if ($this->_topic !== 'getbuyers') {
```

Comment:
instead of negative check, check for the specific method in which we want to add these params

Reply:
Updated PR